translate/storage/properties.py

```diff
@@ -722,6 +731,12 @@ def __init__(self, *args, **kwargs):
         kwargs['encoding'] = "utf-8"
         super(stringsutf8file, self).__init__(*args, **kwargs)
 
+    def is_line_continuation(self, line):
+        l = line.rstrip()
+        if l and l[-1] == ';':
+            return False
+        return True
+
```

Comment:
I don't like the code duplication here, I think this class should inherit from stringsfile to avoid that.

Reply:
Indeed. 
But I will have to change `stringsfile`'s __init__ implementaion because it overrides the given `kwarg['personality']` param.

Reply:
True, some changes to __init__ will be needed as well.

Reply:
PTAL
Tell me if there is something else that should be changed.

(I did not put the `def is_line_continuation` function inside the `propfile` class, to avoid breaking other code that might depend on this function).